frontend/src/components/lib/Gallery/gallery.module.scss

```
@@ -0,0 +1,101 @@
+.Gallery {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: var(--size-13);
```

```suggestion
    grid-template-rows: var(--size-13) var(--size-13);
```

Il faut que tu définisse la taille des deux lignes ici. Sinon une seule ligne va faire cette taille

---

```suggestion
    grid-auto-rows: var(--size-13);
```

Sinon tu peux utiliser `grid-auto-rows` qui définie la taille de toutes les ligne de la grid

--- 

```suggestion
    grid-template-rows: 1fr 1fr;
```

Mais dans le cas actuel, il vaut mieux que tu définisse chaque ligne comme étant de taille `1fr`. Comme ça chaque ligne prendra la moitié de la hauteur du composant.

Grace a ça, le composant aura toujours un ratio de `1fr 1fr 1fr 1fr` sur `1fr 1fr` (soit 2/1)